jvm-packages/xgboost4j/src/test/scala/ml/dmlc/xgboost4j/scala/TrainTestData.scala

```diff
@@ -14,12 +14,14 @@
  limitations under the License.
  */
 
-package ml.dmlc.xgboost4j.scala.spark
+package ml.dmlc.xgboost4j.scala
```

Comment:
if you move this file to xgboost4j, there is a significant part duplicate with TrainTestData.java and Classification.java

additionally, most of methods here are only called by spark package, why we need to move?

Reply:
This is a good point. I remember that I moved it because there was at least one test in xgboost4j which used this data but now I can't seem to figure out which one.

So I will probably try to move it back.